src/Events/Events.jsx

```diff
@@ -103,150 +88,27 @@ export class Events extends Component {
-			descriptionElem = (
-				<Typography
-					component="p"
-					dangerouslySetInnerHTML={{ __html: cleanDescription }}
-				/>
-			);
-		}
-		return (
-			<Card
-				key={mEvent.time}
-				className={mEvent.withDescription ? classes.cardBig : classes.card}
-			>
-				<CardMedia
-					className={classes.media}
-					image={mEvent.coverUrl}
-					title={`${formattedDate} - ${mEvent.name}`}
-				/>
-				<CardContent className={classes.content}>
-					<Typography gutterBottom variant="headline" component="h2">
-						{formattedDate} - {mEvent.name}
-					</Typography>
-					{descriptionElem}
-				</CardContent>
-				<CardActions>
-					<Button size="small" color="primary" href={mEvent.link}>
-						Read More
-					</Button>
-				</CardActions>
-			</Card>
-		);
-	}
-
-	renderHeader(mEvent) {
-		const { t, classes } = this.props;
-		const locale = i18n.language === 'nl' ? 'nl-NL' : 'en-US';
-		const formattedDate = new Date(mEvent.time).toLocaleDateString(locale, {
-			year: 'numeric',
-			month: 'long',
-			day: 'numeric',
-		});
+	render() {
+		const { t } = this.props;
+		const pastEvents = this.state.pastEvents.map(mEvent => (
+			<EventCard key={mEvent.time} MeetupEvent={mEvent} />
+		));
 		return (
-			<div key={mEvent.name}>
-				<section className={classes.section}>
-					<ResponsiveImage
-						path="/images/events/2017-09-28%20Andre%20Staltz%20RxJS.jpg"
-						asBackgroundImage
-						effect="e_art:fes"
-						alt="Andre Staltz presenting to a crowd at the Codestar Night meetup of September of 2018"
-					/>
-					<Container fullHeight center>
-						<div className="row">
-							<div className="col-12">
-								<div className="mt-4">
-									<Typography
-										align="center"
-										variant="display1"
-										style={{ color: 'white' }}
-									>
-										{t('OUR_NEXT_EVENT')}
-									</Typography>
-									<Hidden mdUp>
-										<Typography
-											align="center"
-											variant="display2"
-											className={classes.nextEventTitle}
-										>
-											{mEvent.name}
-										</Typography>
-									</Hidden>
-									<Hidden smDown>
-										<Typography
-											align="center"
-											variant="display4"
-											className={classes.nextEventTitle}
-										>
-											{mEvent.name}
-										</Typography>
-									</Hidden>
-									<Typography
-										gutterBottom
-										align="center"
-										variant="display2"
-										style={{ color: 'white' }}
-									>
-										{formattedDate}
-									</Typography>
-									<div style={{ textAlign: 'center' }}>
-										<Button
-											color="primary"
-											variant="raised"
-											href={mEvent.link}
-											className="mr-1"
-										>
-											{t('SIGN_UP')}
-										</Button>
-										<Button variant="contained" href={mEvent.link}>
-											{t('MORE_INFO')}
-										</Button>
-									</div>
+			<Fragment>
+				<EventsHeader data={this.state.nextEvent} />
```

Comment:
I really like the fact that you've put all the header-related stuff inside the `EventsHeader` component 👍

Reply:
A nice next step would be to also do this for the past events.

Reply:
I don't know if I misunderstand, but I already have an EventCard for this. Or do you mean to make a component for this bit:
```
		const pastEvents = this.state.pastEvents.map(mEvent => (
			<EventCard key={mEvent.time} MeetupEvent={mEvent} />
		));
```